setup.cfg

```diff
@@ -1,9 +1,10 @@
 [flake8]
-exclude = .git,.env,venv,__pycache__,.eggs
+exclude = .git,.env,venv,__pycache__,.eggs,{{cookiecutter.project_slug}}
```

Comment:
hmm, I am not sure: do we really need to exclude user's code directory from linters?

Reply:
Achso, this is our setup.cfg, not the template's. Keeping changes.